repo2docker/buildpacks/conda/environment.frozen.yml

```diff
@@ -1,102 +1,116 @@
 # AUTO GENERATED FROM environment.py-3.7.yml, DO NOT MANUALLY MODIFY
-# Frozen on 2019-09-13 19:50:21 UTC
+# Frozen on 2020-07-25 17:16:14 UTC
 name: r2d
 channels:
   - conda-forge
   - defaults
   - conda-forge/label/broken
 dependencies:
-  - _libgcc_mutex=0.1=main
-  - alembic=1.1.0=py_0
-  - asn1crypto=0.24.0=py37_1003
+  - _libgcc_mutex=0.1=conda_forge
+  - _openmp_mutex=4.5=0_gnu
+  - alembic=1.4.2=pyh9f0ad1d_0
   - async_generator=1.10=py_0
-  - attrs=19.1.0=py_0
-  - backcall=0.1.0=py_0
-  - bleach=3.1.0=py_0
+  - attrs=19.3.0=py_0
+  - backcall=0.2.0=pyh9f0ad1d_0
+  - bleach=3.1.5=pyh9f0ad1d_0
   - blinker=1.4=py_1
-  - bzip2=1.0.8=h516909a_1
-  - ca-certificates=2019.9.11=hecc5488_0
-  - certifi=2019.9.11=py37_0
+  - brotlipy=0.7.0=py37h8f50634_1000
+  - ca-certificates=2020.6.20=hecda079_0
+  - certifi=2020.6.20=py37hc8dfbb8_0
   - certipy=0.1.3=py_0
-  - cffi=1.12.3=py37h8022711_0
-  - chardet=3.0.4=py37_1003
-  - configurable-http-proxy=4.1.0=node11_1
-  - cryptography=2.7=py37h72c5cf5_0
-  - decorator=4.4.0=py_0
-  - defusedxml=0.5.0=py_1
-  - entrypoints=0.3=py37_1000
-  - idna=2.8=py37_1000
-  - ipykernel=5.1.2=py37h5ca1d4c_0
-  - ipython=7.8.0=py37h5ca1d4c_0
+  - cffi=1.14.0=py37hd463f26_0
+  - chardet=3.0.4=py37hc8dfbb8_1006
+  - cryptography=3.0=py37hb09aad4_0
+  - decorator=4.4.2=py_0
+  - defusedxml=0.6.0=py_0
+  - entrypoints=0.3=py37hc8dfbb8_1001
+  - idna=2.10=pyh9f0ad1d_0
+  - importlib-metadata=1.7.0=py37hc8dfbb8_0
+  - importlib_metadata=1.7.0=0
+  - ipykernel=5.3.4=py37h43977f1_0
```

Comment:
Do we need these `ipython` and `jupyter` stuff? I thought we removed this already 😅